scvi/model/_scvi.py

```diff
@@ -163,3 +164,24 @@ def setup_anndata(
             continuous_covariate_keys=continuous_covariate_keys,
             copy=copy,
         )
+
+    @setup_anndata_dsp.dedent
+    def anndata_fields(batch_key=None, labels_key=None, layer=None):
```

Comment:
I think this is a bit much for devs. Let's try the decorator as we discussed on slack